resource-managers/kubernetes/integration-tests/tests/py_container_checks.py

```diff
@@ -24,7 +24,7 @@ def version_check(python_env, major_python_version):
         These are various tests to test the Python container image.
         This file will be distributed via --py-files in the e2e tests.
     """
-    env_version = os.environ.get('PYSPARK_PYTHON')
+    env_version = os.environ.get('PYSPARK_PYTHON', 'python3')
```

Comment:
This looks safe, but when this default value is used? I guess `PYSPARK_PYTHON` exists always. Is it just a fail-safe or an existing bug?

Reply:
Oh, when Spark application is submitted via `spark-submit`, looks its possible its unset. Without this change, the test fails.